components/Input/ListSelection/index.js

```diff
@@ -82,6 +90,30 @@ export class NormalListSelection extends React.PureComponent {
         return className;
     }
 
+    getFilteredOptions = memoize((searchText, options) => {
+        const {
+            labelSelector,
+        } = this.props;
+
+        if (isFalsyString(searchText)) {
+            return options;
+        }
+
+        const newOptions = options
+            .filter(
+                option => (
+                    searchText === undefined
```

Comment:
Checking `undefined` is not needed for function `caseInsensitiveSubmatch`
I think!